mirgecom/diffusion.py

```diff
@@ -212,7 +306,7 @@ class _DiffusionGradTag:
 
 def grad_operator(
         dcoll, boundaries, u, *, quadrature_tag=DISCR_TAG_BASE, dd=DD_VOLUME_ALL,
-        comm_tag=None):
+        comm_tag=None, **kwargs):
```

Comment:
Pretty please no `**kwargs`. 🙂 It's kind of an abuse of that functionality the way we're using it.